bigquery/integration_test.go

```diff
@@ -54,6 +54,7 @@ var record = flag.Bool("record", false, "record RPCs")
 
 var (
 	client                 *Client
+	sClient                *Client
```

Comment:
maybe make this name less ambiguous, since we already track a storage (GCS) client in here as well?

Reply:
yeah, I didn't liked the name too. Will make it more explicit.